demo/admin/src/App.tsx

```diff
@@ -100,58 +101,68 @@ export function App() {
-                                                                            <Route
-                                                                                path={`${match.path}/preview`}
-                                                                                render={(props) => (
-                                                                                    <SitePreview
-                                                                                        resolvePath={(path: string, scope) => {
-                                                                                            return `/${scope.language}${path}`;
-                                                                                        }}
-                                                                                        {...props}
-                                                                                    />
-                                                                                )}
-                                                                            />
-                                                                            <Route
-                                                                                render={() => (
-                                                                                    <MasterLayout
-                                                                                        headerComponent={MasterHeader}
-                                                                                        menuComponent={AppMasterMenu}
-                                                                                    >
-                                                                                        <MasterMenuRoutes menu={masterMenuData} />
-                                                                                    </MasterLayout>
-                                                                                )}
-                                                                            />
-                                                                        </Switch>
-                                                                    )}
-                                                                </ContentScopeProvider>
-                                                            </RouterBrowserRouter>
-                                                        </CurrentUserProvider>
-                                                    </CmsBlockContextProvider>
-                                                </SnackbarProvider>
-                                            </DndProvider>
-                                        </MuiThemeProvider>
+                                        <BlocksConfigProvider
+                                            supportsBlock={(name, scope) => {
```

Comment:
I'm unsure about the name. Maybe we should change it to `isBlockAllowed`. What do you think?

Reply:
@nsams @thomasdax98 what do you think?

Reply:
41ae472cc468b813f04b8ecc5d0ce253eb03b5b4

Reply:
maybe "filterBlock"?

Reply:
Wouldn't filterBlock implicate that the function returns a list of supported blocks?

Reply:
I thought it to be like Array.filter

Reply:
I've considered renaming it to filterBlock but decided against it because it then isn't clear what the return value indicates. If filterBlock returns true it could be understood as "the block should be filtered/removed", since the Array context isn't present.

However, based on your suggestion I changed the argument from the block name to the block itself: 748e9acea03e0f34cec98529f46d183052333719. This allows additional methods for filtering block, e.g., filter all blocks from a specific category.